metric/cpu/metric_windows_test.go

```diff
@@ -0,0 +1,51 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//go:build windows
+
+package cpu
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCounterLength(t *testing.T) {
+	query, err := buildQuery()
```

Comment:
Isn't it better to rely on `var query, qError = buildQuery()` instead of calling buildQuery explicitly? This way we make sure that the globals used by the real library code works as intended. This guards against any issue that can happen during initialization. Does that make sense here?

Reply:
Additionally, I'm not sure if this has any impact but wouldn't be better to call buildQuery inside init()? I think globals are initialized before init, but I might be wrong.

Reply:
> Additionally, I'm not sure if this has any impact but wouldn't be better to call buildQuery inside init()

That would look better. I'll update the PR. Thank!

> Isn't it better to rely on var query, qError = buildQuery() instead of calling buildQuery explicitly

I'll move the `var query, qError = buildQuery()` under `init()`.